example/src/main/java/org/aerogear/mobile/example/ExampleApplication.java

```diff
@@ -17,9 +21,36 @@ public void onCreate() {
         mobileCore = MobileCore.init(this);
         metricsService = mobileCore.getInstance(MetricsService.class);
 
+        sendDefaultMetrics();
```

Comment:
I wonder if we can make the default metrics sending part of MobileCore.init

Reply:
See: https://github.com/aerogear/aerogear-app-metrics/pull/41

Reply:
There is a discussion about it on ML https://groups.google.com/forum/#!topic/aerogear/QZn65_qglSU